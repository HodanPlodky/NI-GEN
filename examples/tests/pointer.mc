void f(int * p) {
    *p = *p + 1;
}

int main() {
    int a = 1;
    int * p;
    p = &a;
    *p = 10;
    f(&a);
    return a;
}
