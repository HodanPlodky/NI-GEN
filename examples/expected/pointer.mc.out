input file : examples/tests/pointer.mc
return 11
